### DISCUSSION TITLE
Ledger no pin for firefly can I restore using my ledger

### DISCUSSION THREAD

**Discussion by bazrain** [iotaledger/firefly#8051] [Help & Support]
I'm using a ledger with my firefly wallet and have 2 wallets showing only have 1 pin written down it opens one wallet no iota there the other wallet will not open with the 1pin I have....can I restore my wallet using my ledger and setting up a new pin somehow....Thanks in advance

**Comment by begonaalvarezd** [comment]
Yes you can always restore using your ledger, you can set a new pin in firefly to access the new restored profile
